src/biokbase/narrative/tests/data/ee2_job_test_data.json

```diff
@@ -292,7 +292,7 @@
         "widget_info": {
             "name": "no-display",
             "params": {
-                "obj_ref": "18836/5/1",
+                "obj_ref": "62425/2/12",
```

Comment:
These were put in place (I think) because the mocker was switching them around to the one true mock UPA.